ansible_collections/arista/avd/docs/how-to/first-cvp-avd-project.md

```diff
@@ -418,9 +419,9 @@ Go to _Provisioning > Change Control_ to create a new change control
 
 This change control is an example and you are free to build structure you want. In this scenario, all tasks can be run in parallel as we just rollout an EVPN/VXLAN fabric.
 
-## Analyze result
+### Analyze result
 
-Once devices rebooted, you can review fabric status on devices themselfs or on on CloudVision as well.
+Once devices rebooted, you can review fabric status on devices themselves or on on CloudVision as well.
```

Comment:
Could be rephrased to "Once the devices have rebooted, you can verify the fabric status on devices themselves or on CloudVision."